### DISCUSSION TITLE
[Prettier Plugin]  Is it possible to run tailwind class sorting without prettier?

### DISCUSSION THREAD

**Discussion by Nefcanto** [tailwindlabs/tailwindcss#9273] [Help]
We would love to use tailwind's opinionated class order. But we can not use prettier and its opinionated code formatting does not suit our internal policies.

Is it possible for us to use tailwind's class sorting without using prettier?

**Comment by smithliam** [comment]
At this time it only works with Prettier, because it's a Prettier plugin.

**Comment by airblade** [comment]
I wrote a standalone script to do this: https://github.com/airblade/tcs.

**Comment by svoop** [comment]
There's also [Rustywind](https://github.com/avencera/rustywind).

**Comment by saiballo** [comment]
I don't understand why I am forced to use such an invasive tool like Prettier. It's fine to sort CSS classes because that's useful, but Prettier also modifies the spacing in my HTML code, and I can't prevent it.

It's really frustrating not being able to just sort Tailwind classes without other side effects. The choice that has been made is too invasive.
